transportpipes-core/src/main/java/de/robotricker/transportpipes/commands/TPCommand.java

```diff
@@ -61,8 +62,9 @@ public void onTPS(CommandSender cs) {
             Map<BlockLocation, Duct> ductMap = globalDuctManager.getDucts(world);
             for (Duct duct : ductMap.values()) {
                 if (duct.getDuctType().getBaseDuctType().is("Pipe")) {
+                	Pipe pipe = (Pipe) duct;
```

Comment:
Indentation's messed up here, but this should probably be inlined anyway since it's only used once two lines below.

Reply:
Done